translations/en/transformer-handbook.md

```diff
@@ -879,9 +879,77 @@ const visitor = (node: ts.Node): ts.Node => {
 };
 ```
 
-#### Check if an identifier is referenced
+#### Check if two identifiers refer to the same symbol
 
-> **TODO** - Is this possible?
+Identifiers are created by the parser and are always unique.
+Say, if you create a variable `foo` and use it in another line, it will create 2 separate identifiers with the same text `foo`.
+
+Then, the linker runs through these identifiers and connects the identifiers referring to the same variable with a common symbol (while considering scope and shadowing). Think of symbols as what we intuitively think as variables.
+
+So, to check if two identifiers refer to the same symbol - just get the symbols related to the identifier and check if they are the same (by reference).
```

Comment:
if an identifier is _not_ referenced `typeChecker.getSymbolAtLocation(node)` would return `undefined` yeah? should we mention that? 😄